notebooks/process_demetr_results.ipynb

```diff
@@ -0,0 +1,3162 @@
+{
```

Comment:
<p>In terms of the report I think anything about differences between our implementation and the original DEMETR results can be demoted to an appendix or left out completely with a mention/link back to the repo for more details (i.e. just keep the one on the right?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/ARC-m4st/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>